cmd/notifiers.go

```diff
@@ -221,3 +222,16 @@ func filterRuleMatch(object string, frs []filterRule) bool {
 	}
 	return prefixMatch && suffixMatch
 }
+
+// A type to represent dynamic error maker functions for
+// notifications.
+type notificationErrorMaker func(string, ...interface{}) error
```

Comment:
This should be called  `notificationErrorFactoryFunc` instead. Factory is a more common word than Maker.